make/autoconf/build-aux/config.guess

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 #
-# Copyright (c) 2012, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2012, 2021, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2021, Azul Systems, Inc. All rights reserved.
```

Comment:
Will this be removed when upstreamed?

Reply:
I don't expect it to be, no - if you grep through `develop` source for Azul Systems, Inc. there are a few instances dating back to 2016